goi18n/merge_command.go

```diff
@@ -152,6 +157,17 @@ func (mc mergeCommand) marshal(v interface{}) ([]byte, error) {
 	switch mc.format {
 	case "json":
 		return json.MarshalIndent(v, "", "  ")
+	case "toml":
+		m, ok := v.(map[string]interface{})
+		if !ok {
+			return nil, fmt.Errorf("invalid format for marshaling to TOML")
+		}
+		tree, err := toml.TreeFromMap(m)
+		if err != nil {
+			return nil, err
+		}
+		s, err := tree.ToTomlString()
+		return []byte(s), err
```

Comment:
I would pull this logic out into a `marshalTOML(v interface{}) ([]bytes, error)` function.  Or submit a PR upstream.